src/components/Pagination/index.tsx

```diff
@@ -0,0 +1,226 @@
+import classNames from 'classnames';
+import { FC, PropsWithChildren } from 'react';
+export type PaginationProps = PropsWithChildren<{
+  className?: string;
+  currentPage: number;
+  totalPages: number;
+  onPageChange: (page: number) => void;
+  // Optional props
+  showIcon?: boolean;
+  showIconOnly?: boolean;
```

Comment:
Here is just a suggestion:

Instead of using `showIconOnly` and `showOnlyPreviousAndNext`, that I think can cause some confusion. I would prefer to have something like `displayFormat`, where you could pick between `pagination | navigation | navigation-group`. And them use the `showIcon` as a modifier to add the icon or not.

It probably would cover all the styles documented on original flowbite docs.

Reply:
okay thanks @rluders. I will make this changes.

Reply:
done